browsers/file-io.mdx

```diff
@@ -221,7 +226,122 @@ async def main():
 
 if __name__ == "__main__":
     asyncio.run(main())
+````
+
+```typescript Stagehand v3
```

Comment:
Can we move Stagehand v3 out of this section? My thought: 
Outer section for Downloads
Sub-section for Playwright
Sub-section for Stagehand v3

Reply:
See here for inspiration of what I am looking for: https://tbd-6fc993ce-feat-improve-io-docs.mintlify.app/integrations/laminar#browser-agent-framework-examples